truss-chains/truss_chains/definitions.py

```diff
@@ -393,6 +397,7 @@ class MyChainlet(chains.ChainletBase):
     assets: Assets = Assets()
     name: Optional[str] = None
     options: ChainletOptions = ChainletOptions()
+    runtime: Runtime = Runtime()
```

Comment:
Chains doesn't map 1:1 to the structure of truss config. I wonder if it would make sense to add this field to `ChainletOptions` instead of creating a new class.

If you want to proceed with a new class, then there should be some differentiator, explaining which future fields go into `Runtime` and which ones go into `ChainletOptions`.